operator/pkg/manifest/installer.go

```diff
@@ -638,6 +638,92 @@ func waitForCRDs(objects object.K8sObjects, stdout string, dryRun bool) error {
 	return nil
 }
 
+func waitForResources(objects object.K8sObjects, cs kubernetes.Interface) (bool, []string, error) {
```

Comment:
the diff here is bad, its all copy+pasted - no new code. Just breaking out into a method so we can call it in two places